client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1529,35 +1529,22 @@ def _before_repre_load(self, placeholder, representation):
 
         pass
 
-    def _reduce_last_version_repre_entities(self, representations):
+    def _reduce_last_version_repre_entities(self, repre_contexts):
         """Reduce representations to last verison."""
```

Comment:
Since you're changing surrounding code anyway.
```suggestion
        """Reduce representations to last version."""
```